### PR TITLE
User list : User name and Display name visibility improvement

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
Signed-off-by: Jérôme Herbinet jerome.herbinet@arawa.fr

Signed-off-by: Jérôme Herbinet <33763786+Jerome-Herbinet@users.noreply.github.com>

### CODE REVIEW

**Comment by Jerome-Herbinet** [comment]
@szaimen (following our conversation a few minutes ago) I've closed my old messy PR (#33669) and I've created this brand new one, with exactly the same changes. It of course takes in account the design feedback of @jancborchardt. I think that everything should be OK.

**Comment by szaimen** [comment]
> @szaimen (following our conversation a few minutes ago) I've closed my old messy PR (#33669) and I've created this brand new one, with exactly the same changes. It of course takes in account the design feedback of @jancborchardt. I think that everything should be OK (hoping it's not too late for Nextcloud 25).

Thank you! Yes, the PR looks good to me. However I fear we need the compiled js. Can you please run `npm run build` and commit the changes to this branch?

**Comment by Jerome-Herbinet** [comment]
> > @szaimen (following our conversation a few minutes ago) I've closed my old messy PR (#33669) and I've created this brand new one, with exactly the same changes. It of course takes in account the design feedback of @jancborchardt. I think that everything should be OK (hoping it's not too late for Nextcloud 25).
> 
> Thank you! Yes, the PR looks good to me. However I fear we need the compiled js. Can you please run `npm run build` and commit the changes to this branch?

OK @szaimen, it's done.

**Comment by szaimen** [comment]
> OK @szaimen, it's done.

Thanks! However a `npm run sass` and commiting the changed files is unfortunately needed as well. Afterwarsa the node test should be green.

File: apps/settings/src/components/UserList.vue [hunk 1]
```diff
@@ -156,11 +156,12 @@
 			class="row">
 			<div id="headerAvatar" class="avatar" />
 			<div id="headerName" class="name">
-				{{ t('settings', 'Username') }}
-
 				<div class="subtitle">
-					{{ t('settings', 'Display name') }}
+					<strong>
+						{{ t('settings', 'Display name') }}
+					<strong>
```
Comment (szaimen) [inline apps/settings/src/components/UserList.vue:162]:
```suggestion
					</strong>
```
This should fix ESLint. Afterwards please run `npm run build` again and push the changes

Comment (Jerome-Herbinet) [reply apps/settings/src/components/UserList.vue]: Oups, sorry, and fixed (+ build done)

**Comment by szaimen** [comment]
Thanks! I'll review and test it out later on!

**Comment by szaimen** [comment]
This is how it looks now: 
image

**Review by szaimen — APPROVED** [review]
LGTM

**Review by Pytal — APPROVED** [review]

**Review by nimishavijay — APPROVED** [review]
Looks great! :)

**Comment by szaimen** [comment]
CI failure unrelated

**Comment by szaimen** [comment]
Thanks for you patience and effort @Jerome-Herbinet !

**Pull request merged**